project-base/storefront/pages/_app.tsx

```diff
@@ -35,26 +29,42 @@ process.on('uncaughtException', (error: Error, origin: unknown) =>
     }),
 );
 
+const UrqlWrapper = dynamic(() => import('components/Layout/UrqlWrapper').then((component) => component.UrqlWrapper), {
+    ssr: true,
```

Comment:
No need for specify `ssr: true`. This is default value.

Reply:
Thanks, removed for all within the file